slippi-stats.js

```diff
@@ -42,6 +50,19 @@ if (character_arg) {
     character_requested = checkCharacter(character_arg)
 }
 
+function loadCache() {
+    try {
+        const contents = fs.readFileSync(cacheFilePath, 'utf8')
+        const data = JSON.parse(contents)
+        if (!data) { return }
+        if (data.statsVersion != statsVersion) { return }
+        if (data.slippiJsVersion != slippiJsVersion) { return }
+        return data.results
+    } catch {
+        return undefined
```

Comment:
Any reason you're returning `undefined` here, but doing an empty return a few lines above? Functionally they're the same, but for consistency's sake they might as well be the same ?